Containers/imaginary/Dockerfile

```diff
@@ -1,2 +1,15 @@
 # From https://github.com/h2non/imaginary/blob/master/Dockerfile
 FROM nextcloud/imaginary:20220822
+
+USER root
+RUN set -ex; \
+    \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        curl \

```

Comment:
So curl is not in the image?

Reply:
![grafik](https://user-images.githubusercontent.com/75573284/186661531-b71401da-094f-4041-b6f3-cc9a7ddc6913.png)

Reply:
all right!